COMPILING.md

```diff
@@ -87,9 +87,17 @@ git clone --recursive https://github.com/cuberite/cuberite.git
 
 ### Building ###
 
-Follow the instructions at [CMake on Unix-based platforms](#cmake-on-unix-based-platforms).
+Run the following commands to build Cuberite:
+
+```
+mkdir Release
+cd Release
+cmake -DCMAKE_BUILD_TYPE=RELEASE ..
+make -j`nproc`
```

Comment:
The [manual](https://www.gnu.org/software/make/manual/html_node/Parallel.html) says "By default, there is no load limit.". Wouldn't `make -j` be dangerous on small machines like the Pi?